mlflow/models/evaluation/base.py

```diff
@@ -496,33 +634,17 @@ def evaluate(
                              a nested dictionary whose key is the evaluator name.
     :return: An :py:class:`mlflow.models.evaluation.EvaluationDataset` instance containing
              evaluation results.
+
+    The default evaluator supports the 'regressor' and 'classifer' `model_type`s. The available
+    `evaluator_config` options for the default evaluator include:
+     - log_model_explainability: The number of rows to use for calculating model explainability.
```

Comment:
I think this docstring is wrong. Isn't `explainability_nsamples` the target number of rows to use? Shouldn't this be a boolean flag for whether to log the results?